designer/client/src/components/graph/node-modal/properties.tsx

```diff
@@ -65,6 +68,17 @@ export function Properties({
                 setProperty={setProperty}
                 errors={errors}
             />
+            <NodeField
+                isEditMode={isEditMode}
+                showValidation={showValidation}
+                node={node}
+                renderFieldLabel={renderFieldLabel}
+                setProperty={setProperty}
+                errors={errors}
+                fieldType={FieldType.checkbox}
+                fieldName={"additionalFields.showDescription"}
+                description={"Show description every time"}
```

Comment:
Version from mockups sounds more meaningful https://www.figma.com/design/eOiVhy9jX6DLYlG7p6IOFu/Nu-Product-Design?node-id=2715-30367&t=1KZBs5rTVypEthi6-0. Can we adjust this text?